daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -590,6 +590,9 @@ func (devices *DeviceSet) createFilesystem(info *devInfo) error {
 		devices.filesystem = determineDefaultFS()
 	}
 
+	logrus.Infof("devmapper: Creating filesystem %s on device(%s). START", devices.filesystem, info.Name())
+	defer logrus.Infof("devmapper: Creating filesystem %s on device(%s). END", devices.filesystem, info.Name())
```

Comment:
potentially minor, but with a `defer` executed log entry, that means that even in all error cases you get the log entry, even if no actual filesystem creation happened?  Overall this seems like a good thing to log, but wonder if it should be more clearly aligned with actual creation starting and finishing..